src/ui/authenticationInApp/ViewAuthenticationSsoAccount.qml

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.5
+import QtQuick.Layouts 1.14
+
+import Mozilla.Shared 1.0
+import Mozilla.VPN 1.0
+import components 0.1
+import components.inAppAuth 0.1
+
+MZInAppAuthenticationThirdParty {
+    id: authSsoAccount
+
+    _telemetryScreenId: "authentication_sso_account"
+    _viewObjectName: "authSsoAccount"
+    _imgSource: "qrc:/ui/resources/create-password.svg"
+    _headlineText: "Create a password"
```

Comment:
this needs to be added to strings.yaml

also, not sure if this is intentional, but the desktop and mobile versions have different headlines